mountpoint-s3/src/inode.rs

```diff
@@ -334,11 +328,17 @@ impl Superblock {
             return Err(InodeError::FileAlreadyExists(inode.err()));
         }
 
-        // Local inode stats never expire, because they can't be looked up remotely
         let stat = match kind {
             // Objects don't have an ETag until they are uploaded to S3
-            InodeKind::File => InodeStat::for_file(0, OffsetDateTime::now_utc(), None, None, None, NEVER_EXPIRE_TTL),
-            InodeKind::Directory => InodeStat::for_directory(self.inner.mount_time, NEVER_EXPIRE_TTL),
+            InodeKind::File => InodeStat::for_file(
+                0,
+                OffsetDateTime::now_utc(),
+                None,
+                None,
+                None,
+                self.inner.cache_config.file_ttl,
```

Comment:
Maybe this will never happen since I'm guessing applications almost always open the file immediately after performing a `mknod`, but an `open` 100ms after the `mknod` will now go to S3 even though we know we'll always choose the local node over a remote one.

Not worried right now, but just noting this observation.